notebooks/en/finetune_t5_for_search_tag_generation.ipynb

```diff
@@ -1,1063 +1,1666 @@
 {
```

Comment:
<p><code>os</code> is not imported here so it'll generate an error. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>